openssl-dynamic/src/main/c/ssl.c

```diff
@@ -1018,10 +984,9 @@ TCN_IMPLEMENT_CALL(void, SSL, freeSSL)(TCN_STDARGS,
     tcn_ssl_ctxt_t* c = NULL;
     tcn_ssl_verify_config_t* verify_config = NULL;
     SSL *ssl_ = J2P(ssl, SSL *);
-    if (ssl_ == NULL) {
-        tcn_ThrowException(e, "ssl is null");
-        return;
-    }
+
+    TCN_CHECK_NULL(ssl_, "ssl is null",);
```

Comment:
consider clarifying this as follow: `TCN_CHECK_NULL(ssl_, "ssl is null", /*void*/);`